packages/plugins/test/mocha/foundry/utils.spec.ts

```diff
@@ -814,7 +814,9 @@ describe('Utils', async () => {
           projectRoot
         )
       ).to.be.rejectedWith(
-        `Detected linked library in: ${sourceName}:MyLinkedLibraryContract\n` +
+        `Detected linked library in: ${resolve(
```

Comment:
Since foundry outputs build info files with absolute paths in our repo, we have to update this test to check for that.